browser/ui/webui/brave_rewards/rewards_page_data_source.cc

```diff
@@ -25,9 +25,12 @@ static constexpr webui::ResourcePath kResources[] = {
     {"favicon.ico", IDR_BRAVE_REWARDS_FAVICON}};
 
 static constexpr webui::LocalizedString kStrings[] = {
-    {"appError", IDS_REWARDS_APP_ERROR_TITLE},
+    {"appErrorTitle", IDS_REWARDS_APP_ERROR_TITLE},
     {"cancelButtonLabel", IDS_REWARDS_PANEL_CANCEL},
     {"closeButtonLabel", IDS_BRAVE_REWARDS_ONBOARDING_CLOSE},
+    {"connectAccountSubtext", IDS_REWARDS_CONNECT_ACCOUNT_TEXT_2},
+    {"connectAccountText", IDS_REWARDS_CONNECT_ACCOUNT_SUBTEXT},
```

Comment:
Seems a little confusing that subtext maps to text and vice versa.

Reply:
Yep - that's not quite right 😄

Reply:
Fixed.